other-js/a11y.js

```diff
@@ -7,3 +7,13 @@ function announce(message) {
 function announceSlideNumber(slideNumber) {
   announce("Now on slide " + (currentSlideIndex + 1));
 }
+
+_2DNote.setAs2DArea(document.documentElement, callbackUponUpdate);
+
+function callbackUponUpdate(e) {
+  var tagName = document.activeElement.tagName;
+  var isText = tagName === "P";
+  var isImage = tagName === "IMG";
+  console.log(isText || isImage);
+  if (!isText && !isImage) _2DNote.stop();
```

Comment:
i later decided against doing it this way

Reply:
because this is slightly different from adding a listener on the document and detecting the element type